var/spack/repos/builtin/packages/nettle/package.py

```diff
@@ -24,6 +24,7 @@ class Nettle(AutotoolsPackage, GNUMirrorPackage):
 
     depends_on("gmp")
     depends_on("m4", type="build")
+    depends_on("openssl")
```

Comment:
Is this the case for all listed versions of from a specific one on?

Reply:
Yes, nettle is in the crypto domain.